data.sql

```diff
@@ -5,4 +5,11 @@ VALUES
 ('Gabumon','2018-11-15',2,8,true), 
 ('Pikachu','2021-01-07',1,15.04,false),
 ('devimon','2017-05-12',5,11,true),
+('Charmander','2020-02-08',0,11,false),
+('Plantmon' , '2021-11-15',2,5.7,true),
+('Squirtle','1993-04-02',3,12.13,false),
+('Angemon','2005-06-12',1,45,true),
+('Boarmon','2005-06-07',7,20.4,true),
+('Blossom','1998,10-13',3,17,true),
+('Ditto','2022-05-14',4,22,true)
```

Comment:
Some of the weights are supposed to be negative as per project requirements.
e.g 

- `Charmander` is supposed to have a weight of `-11.0`
-`Plantmon` is supposed to have a weight of `-5.7`
- `Squirtle` is supposed to have a weight of `-12.13` and so forth...

Kindly correct all your weights and make sure that all negative weights are inserted.